.github/vale/dicts/aiven.dic

```diff
@@ -28,6 +28,7 @@ CloudWatch
 codecs
 compliances
 Conduktor
+config
```

Comment:
"config" is not a word, please use "configuration" and remove this from the dictionary.

Reply:
@lornajane  Thanks for looking at this. This is only used once in the instructions, but that's because that's how it's written in the UI. Should I still change it to SAML configuration?

> In the **SAML config** section, click **Edit**.

Reply:
Oh that's a great question.... paging @TibsAtWork for a second opinion. We'll end up with "config" everywhere if it's in the dictionary, but if we're using it in the interface?

Reply:
Sounds sensible to me 👍

Reply:
+1 from me on option 3 as well. The sensible options is always my favourite!